### ISSUE TITLE
Accessibility: Critical information disappears after a time interval [Edit Address]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51830]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

[Error message 'Please include an '@' in the email address. 'evamaria adam' is missing an '@''] . This element has critical functionality that disappears quickly before a user may be able to interact with it. Some users may need more time to think, react, or to navigate to the functionality and will be unable to interact with the element before it disappears.

![T006_ErrorDisappears](https://github.com/user-attachments/assets/083afa6a-5d1a-4efc-9676-c0dd6d1a43f0)

### Component or Page: 

Edit Address My Account Page

### Name affected element:

Error message, 'Please include an '@' in the email address. 'evamaria adam' is missing an '@'.

### Relevant code snippets:

`<span class="woocommerce-input-wrapper"><input type="email" class="input-text " name="billing_email" id="billing_email" placeholder="" value="evamaria.adam@example.net" autocomplete="email username"></span>`

### Recommended fix:

Make sure critical , e.g., errors, warnings, or success messages, are always available and do not disappear after a few seconds. It is recommended that these messages stay present on the screen and instead of removing them after a time, you provide a dismiss button so users can choose to close them or they just stay visible until the form reauthenticates. If you want to have messages automatically disappear, you should provide a user preferences setting that allows users to adjust the time messages stay on the screen before disappearing.

**Comment by mikejolley** [comment]
Same feedback as https://github.com/woocommerce/woocommerce/issues/51829 These reports could probably be combined.

**Comment by darrenjacoby** [comment]
Hi @mikejolley , this is default Chrome behaviour based on the field being `required` and marked as `type=email`. Updating this would mean introducing client side validation for forms. This would apply to https://github.com/woocommerce/woocommerce/issues/51829. These forms use server validation and work as expected once submitted.

As an alternative, we can add `novalidate` to form and disable Chrome default validation behaviour. Let me know what's preferable here.

cc @vikrampm1 @dkotter

**Comment by vikrampm1** [comment]
@mikejolley any comment on the above?

**Comment by mikejolley** [comment]
It's frustrating the browser controls are not more accessible by default. 

I would add a JS layer on top to surface the errors in this case; for required inputs also. If there are no prior examples of JS validation on any of the account fields in woo core, copy the pattern of checkout block and place the error underneath the field please.

**Comment by Manussakis** [comment]
> I would add a JS layer on top to surface the errors in this case; for required inputs also.

@mikejolley Could we take another look at this?  I genuinely think adding the `novalidate` attribute to the form, as suggested by @darrenjacoby , is the most suitable solution. These are the reasons:
1. The current server validation works as expected and has already different messages depending on how the email value is submitted:
**Invalid email value**:
![Image](https://github.com/user-attachments/assets/4e42580f-a7ea-4078-a619-0ef9349c75a5)
**Empty email value**:
![Image](https://github.com/user-attachments/assets/f6f40b0c-ccf9-4c60-935a-fca1ef8c837b)
2. The same type of error notice is used in several form on the My Account page
- Edit Billing Address
![Image](https://github.com/user-attachments/assets/4dfc582a-d109-44e2-94ef-4e7e578bee3b)
- Edit Shipping Address
![Image](https://github.com/user-attachments/assets/94838e0d-72dd-493a-abe3-37e5183d473a)
- Edit Account Details
![Image](https://github.com/user-attachments/assets/816d511a-70e1-4f90-bc19-39b19d1bc3fb)
3. Adding JS validation that displays an error message beneath the input could introduce regressions, as it involves adding a new markup to the template.

I tested the https://github.com/woocommerce/woocommerce/pull/54355 PR and it fixed the issue with minimal impact on the code.
Would you mind giving it another thought?

Thanks!

**Comment by mikejolley** [comment]
Ok I'm fine with this @Manussakis as its the simplest fix. We can look into proper inline validation when this UI is blockified in the future.

Thanks
